app/labor/article_suggester.rb

```diff
@@ -24,7 +24,7 @@ def articles(max: 4)
   end
 
   def self.memoized_articles_count
-    @memoized_articles_count ||= Article.published.estimated_count
```

Comment:
Wouldn't this method no longer be considered memoized?

Reply:
good call! fixed!